src/components/SearchResults/UserAccount/UserAccount.js

```diff
@@ -1,19 +1,19 @@
 import styles from "./UserAccount.module.css";
 
-const UserAccount = (props) => {
+const UserAccount = ({ public_repos, followers, following }) => {
   return (
     <div className={styles.userAccount}>
       <p>Repos</p>
       <p>Followers</p>
       <p>Following</p>
       <p className={styles.strong}>
-        <strong>{props.accountInfo.repo}</strong>
+        <strong>{`${public_repos || "-"}`}</strong>
```

Comment:
We can remove the string interpolation in this file. We are only doing a comparison to see what should render. The below suggestion should work fine:
```suggestion
        <strong>{public_repos || "-"}</strong>
```